README.md

```diff
@@ -1,8 +1,48 @@
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_OmairFarooqui/tree/master/core_java)]: Grep App. Designed an application that mimics the Linux grep command with Core Java. The purpose of this application is to search through a given directory for any matches to a user-given string and write the found matching strings to an output file. The technologies used include Java 8, IntelliJ IDE, Maven, Docker, and Git.
+
+
+## Highlighted Projects
+**IoT Smart Home Security System**: Led a team consisting of four Software Engineering students to develop a Smart Home Security System using Arduino, Java, C, ESP8266 Wi-Fi Module, PIR Motion Sensor, Android, Firebase, and ArduCAM Mini. Application offers arm/disarm feature, voice command capability via Amazon Alexa, live video camera footage and capability to record footage. The application was built using the agile design methodology and  tested using the big bang integration testing method. This was a final capstone project and some responsibilities included but were not limited to weekly reports describing work done in the week, weekly meetings with faculty advisor, meetings with group members three times a week, and prototype demonostration.
+
+**ElectionBlock, Blockchain-Based Voting System**: Collaborated with a team of four Software Engineering students to develop a Python permission-based blockchain voting system that integrates a biometric scanner for authentication and proof of authority as a consensus protocol. Developed in Python, the system runs on its own centralized blockchain network. Some of the technologies used include Flask, Redis (Pub-Sub Broker), React, and Raspberry Pi. Published under the 2021 IEEE International Conference on Software Architecture Companion (ICSA-C). (https://bit.ly/3r6TgmA)
+
+**COVID-19 Tracking Mobile Application**: Collaborated with a team of four Software Engineering students to develop a COVID-19 mobile application using Android (Java), which keeps users up- to-date with the latest COVID-19 statistics. The application includes a main page with updated global statistics, the use of Google Maps API to pinpoint a location and display statistics, and a search query function to display statistics for user-specific countries/cities.
+
+**Polling Cloud Web Application**: Developed a polling web application using Python (Flask), HTML, CSS, Amazon RDS (MySQL) and deployed it to an Amazon EC2 instance (IaaS). The application allows users to create their own polls, vote in any poll made by users, and view results of all polls in a table format. All data is stored and accessible using a MySQL database. The database was setup using Amazon RDS (SaaS).
```

Comment:
**Polling Cloud Web Application**: Developed a polling web application using Python (Flask), HTML, CSS, Amazon RDS (MySQL) and deployed it to an Amazon EC2 instance (IaaS). The application allows users to create their own polls, vote in any poll made by users, and view the results of all polls in a table format. All data is stored and accessible using a MySQL database. The database was set up using Amazon RDS (SaaS).